plugin.xml

```diff
@@ -182,35 +182,13 @@
                 <source url="https://cdn.cocoapods.org/"/>
             </config>
             <pods use-frameworks="true">
-                <pod name="FBSDKCoreKit" spec="5.15.0" />
-                <pod name="FBSDKLoginKit" spec="5.15.0" />
-                <pod name="FBSDKShareKit" spec="5.15.0" />
-                <pod name="FBAudienceNetwork" spec="5.6.1" />
+                <pod name="FBSDKCoreKit" spec="7.1.0" />
+                <pod name="FBAudienceNetwork" spec="5.9.0" />
             </pods>
         </podspec>
 
     </platform>
 
-    <!-- browser -->
-    <platform name="browser">
-        <js-module src="www/facebook-browser.js" name="FacebookConnectPluginBrowser">
-            <clobbers target="facebookConnectPlugin" />
-        </js-module>
-        <config-file target="config.xml" parent="/*">
-            <feature name="FacebookConnectPlugin">
-                <param name="browser-package" value="FacebookConnectPluginBrowser"/>
-                <param name="onload" value="true" />
-            </feature>
-            <plugin name="FacebookConnectPlugin" value="FacebookConnectPlugin"/>
-            <access origin="https://m.facebook.com" />
-            <access origin="https://connect.facebook.net" />
-            <access origin="https://graph.facebook.com" />
-            <access origin="https://api.facebook.com" />
-            <access origin="https://*.fbcdn.net" />
-            <access origin="https://*.akamaihd.net" />
-        </config-file>
-    </platform>
-
```

Comment:
This would be good left in for the potential of an electron app, if we are happy to add back in later, then happy for it to go for now.

Reply:
Happy to add it back in later

Reply:
Could we keep this, and do the same as the others and remove the bits we don't use?

Reply:
Ok